fuzz/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "ruff-fuzz"
+version = "0.0.0"
+authors = [
+  "Addison Crump <research@addisoncrump.info>",
+]
```

Comment:
CC: @charliermarsh

Reply:
I leave authorship details to your discretion; this was autofilled by my IDE.

Reply:
The authors field is ["open to interpretation"](https://doc.rust-lang.org/cargo/reference/manifest.html#the-authors-field) but I mostly see it as a mechanism through which users can contact maintainers. Can you add me as well here (`"Charlie Marsh <charlie.r.marsh@gmail.com>"`)? Since I'll be the primarily-responsible maintainer (or at least the point of contact) in the future.

Reply:
(Feel free to include yourself here or remove as you prefer, I don't feel strongly, I'd just like to have myself listed in addition as the primary maintainer.)